ocaml/xapi/xapi_pif.ml

```diff
@@ -443,8 +443,17 @@ let introduce_internal
   (* return ref of newly created pif record *)
   pif
 
+let assert_no_clustering_enabled ~__context ~network =
```

Comment:
You moved this function twice in this PR, could you put it in the correct place in the first commit please?